spark/src/main/kotlin/com/adevinta/spark/components/textfields/SelectTextField.kt

```diff
@@ -281,13 +282,13 @@ public fun SparkSelectTrailingIcon(
     // action. When there's an API to check if Talkback is on, developer will be able to
     // expand the menu on icon click in a11y mode only esp. if using their own custom
     // trailing icon.
-    IconButton(onClick = onIconClick, modifier = modifier.clearAndSetSemantics { }) {
+    IconButton(onClick = onIconClick, modifier = modifier.clearAndSetSemantics { }.requiredSize(24.dp)) {
```

Comment:
Can we add this in IconButton defaults?

Reply:
Actually we don't as this is a special case for the textfield